drivers/common/opensensors.h

```diff
@@ -0,0 +1,30 @@
+#ifndef __OPENSENSORS_H__
+#define __OPENSENSORS_H__
+
+#define MAXSENSORS           10
```

Comment:
NUMSENSORS is defined in sensors.h. It must be set according to the number of sensors supported by the specific board.
OPENSENSORS is defined in opensensors.h and matches the possible sensor types (temperature, humidity, etc.) listed in an enum. The enum contains also a SENSOR_DEFAULT, for possible testing of new sensors, which do not match the other types present into the enum.
In the current implementation, OpenMote-CC2538 defines 7 sensors (so NUMSENSORS in sensors.h is 7), while the opensensors driver is much more general, thus defining a NUMOPENSENSORS that can be higher than those supported by each single board. NUMOPENSENSORS is also the maximum number of CoAP resources that can be registered. Of course, if a board has got 3 sensors, CoAP will show 3 resources.